csv2cmi.py

```diff
@@ -41,12 +41,26 @@
                     help='add line numbers', action='store_true')
 parser.add_argument('--version', action='version',
                     version='%(prog)s ' + __version__)
+parser.add_argument('--delimiter', help='delimiter for different values within cell')
 args = parser.parse_args()
 
 # set verbosity
 if args.verbose:
+    logs.setLevel('DEBUG')
```

Comment:
We changed level to debug because passing `-v` on the command line had no effect with info level. Is this on purpose?

Reply:
Infos are thrown only for uncertain dates and names, like in this example:  
```
key,sender,senderID,senderPlace,senderPlaceID,senderDate,addressee,addresseeID,addresseePlace,addresseePlaceID,addresseeDate,note,edition
I:2,[Johann Sebastian Bach],11850553X,Leipzig,http://www.geonames.org/2879139,1749-12?-27,Graf Wilhelm von Schaumburg-Lippe,118632949,[Bückeburg],http://www.geonames.org/2942159,,,
```

(Debug level is for developing and shouldn't bother the user.)

If it still isn't showing anything, it might be that your "root logger already has handlers configured", see: https://docs.python.org/3.6/library/logging.html#logging.basicConfig

Reply:
To add: this works fine on my Linux and Mac machines.

Reply:
Verbosity is back on info level now. I can't tell if it's due to my root logger configurations or Windows OS that I don't get any info thrown.

Reply:
Please open a separate issue for this (and investigate it further).